error-handlers/error-handler.js

```diff
@@ -1,15 +1,12 @@
 exports.error_handler = (req, res) => {
   res.status(404).send({ msg: "path is not correct" });
 };
-exports.psqlErrors = (error, res) => {
-  console.log("PPSLS");
-  console.log(error);
-  //  if (err.code === "22P02") {
-  //    console.log("PASSED");
-  //    res.status(400).send({ msg: "bad request" });
-  //  }
-};
+exports.psqlErrors = (error, res) => {};
 
-exports.handle_custom_errors = (err, req, res, next) => {
-  console.log("CUSTOM ERROR");
+exports.handle_all_errors = (err, req, res, next) => {
```

Comment:
If you look at the bottom of the error handling notes it will cover how to make the necessary error handlers. It would be nice to see one function for psql errors, one for your own custom rejects and so on. It means you don't have a complicated if/else chain to track through should you have an error code you weren't expecting.